Mlem/Views/Shared/Cached Image.swift

```diff
@@ -14,9 +14,10 @@ import Nuke
 
 struct CachedImage: View {
     
+    @EnvironmentObject var quickLookPaths: QuickLookPaths
```

Comment:
Why is this an `@EnvironmentObject`? It's not used anywhere else--can this be an `@State var quickLookUrl: URL?`?

Reply:
Initial version's dismiss gesture didn't work, so I tried moving the quick look sheet to the window view. No longer necessary, since interactive gesture works if we present QLPreviewController inside another UIKit view controller. 

Moved that to a @StateObject on CachedImage.